dist/images/events/previous-page-arrow.svg

```diff
@@ -0,0 +1 @@
+<svg fill="currentColor" width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" d="M12 4a8 8 0 018 8 8 8 0 01-8 8 8 8 0 01-8-8 8 8 0 018-8zm0 1.958c-3.315 0-6.014 2.71-6.014 6.042S8.685 18.042 12 18.042c3.315 0 6.014-2.71 6.014-6.042S15.315 5.958 12 5.958zm.69 10.1l-3.582-4 3.695-4.117 1.49 1.336-2.5 2.782 2.387 2.665-1.49 1.334z"/></svg>
```

Comment:
We aren't using this SVG. We can actually remove the whole `dist/images/events/` directory.